### PR TITLE
config: try to coerce string values into numbers

### PR DESCRIPTION
**Pull request by Rugvip**

### CODE REVIEW

File: packages/config-loader/src/loader.test.ts [hunk 1]
```diff
@@ -145,25 +145,4 @@ describe('loadConfig', () => {
       },
     ]);
   });
-
-  it('coerces port to a number', async () => {
```
Comment (Rugvip) [inline packages/config-loader/src/loader.test.ts:149]: Removed this since coercion is done at read-time instead of when loading

Comment (taras) [reply packages/config-loader/src/loader.test.ts:149]: Reverted the commit that introduced this test

Comment (taras) [reply packages/config-loader/src/loader.test.ts:149]: 👍

**Review by taras — APPROVED** [review]
👍

**Pull request merged**